sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/OpenTelemetrySdkAutoConfiguration.java

```diff
@@ -1,168 +0,0 @@
-import io.opentelemetry.sdk.metrics.exemplar.ExemplarFilter;
-import io.opentelemetry.sdk.resources.Resource;
-import io.opentelemetry.sdk.trace.SdkTracerProvider;
-import io.opentelemetry.sdk.trace.export.SpanExporter;
-import io.opentelemetry.sdk.trace.samplers.Sampler;
-import java.util.ServiceLoader;
-import java.util.function.BiFunction;
-
-/**
- * Auto-configuration for the OpenTelemetry SDK. As an alternative to programmatically configuring
- * the SDK using {@link OpenTelemetrySdk#builder()}, this package can be used to automatically
- * configure the SDK using environment properties specified by OpenTelemetry.
- *
- * @deprecated Use {@link AutoConfiguredOpenTelemetrySdk}.
- */
-@Deprecated
-public final class OpenTelemetrySdkAutoConfiguration {
-
-  /**
-   * Returns an {@link OpenTelemetrySdk} automatically initialized through recognized system
-   * properties and environment variables.
-   *
-   * <p>This will automatically set the resulting SDK as the {@link
-   * io.opentelemetry.api.GlobalOpenTelemetry} instance.
-   *
-   * @deprecated Use {@link AutoConfiguredOpenTelemetrySdk#initialize()}.
-   */
-  @Deprecated
-  public static OpenTelemetrySdk initialize() {
-    return AutoConfiguredOpenTelemetrySdk.initialize().getOpenTelemetrySdk();
-  }
-
-  /**
-   * Returns an {@link OpenTelemetrySdk} automatically initialized through recognized system
-   * properties and environment variables.
-   *
-   * @param setResultAsGlobal Whether to automatically set the configured SDK as the {@link
-   *     io.opentelemetry.api.GlobalOpenTelemetry} instance.
-   * @deprecated Use {@code
-   *     AutoConfiguredSdk.builder().setResultAsGlobal(setResultAsGlobal).build().getOpenTelemetrySdk()}.
-   */
-  @Deprecated
-  public static OpenTelemetrySdk initialize(boolean setResultAsGlobal) {
-    return AutoConfiguredOpenTelemetrySdk.builder()
-        .setResultAsGlobal(setResultAsGlobal)
-        .build()
-        .getOpenTelemetrySdk();
-  }
-
-  /**
-   * Returns an {@link OpenTelemetrySdk} automatically initialized through recognized properties
-   * contained in the {@code config} parameter.
-   *
-   * @param setResultAsGlobal Whether to automatically set the configured SDK as the {@link
-   *     GlobalOpenTelemetry} instance.
-   * @param config A {@link ConfigProperties} instance that contains properties that are to be used
-   *     to auto-configure the returned {@link OpenTelemetrySdk}.
-   * @deprecated Use {@code
-   *     AutoConfiguredSdk.builder().setResultAsGlobal(setResultAsGlobal).setConfig(config).build().getOpenTelemetrySdk()}.
-   */
-  @Deprecated
-  public static OpenTelemetrySdk initialize(boolean setResultAsGlobal, ConfigProperties config) {
-    return AutoConfiguredOpenTelemetrySdk.builder()
-        .setResultAsGlobal(setResultAsGlobal)
-        .setConfig(config)
-        .build()
-        .getOpenTelemetrySdk();
-  }
-
-  static OpenTelemetrySdk newOpenTelemetrySdk(
```

Comment:
This logic was moved to `AutoConfiguredOpenTelemetrySdk#build()`, and `#configureMeterProvider` was moved to `MeterProviderConfiguration`.